common/tool.c

```diff
@@ -182,17 +182,15 @@ command_usage (const p11_tool_command *commands)
 static void
 verbose_arg (void)
 {
-	putenv ("P11_KIT_DEBUG=tool");
```

Comment:
I wonder why putenv was there in the first place. The only reason I can think of is that putenv works in windows.